<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSysSetting</class>
 <widget class="QTabWidget" name="CSysSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统配置</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tabCommonSetting">
   <attribute name="title">
    <string>常用设置</string>
   </attribute>
   <widget class="QWidget" name="wgCommon" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>361</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>月租卡时间提示</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>128</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>剩余</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>天时提醒</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>工作模式</string>
     </property>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>主机模式</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>从机模式</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>离开时需要确认放行</string>
     </property>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>计时卡</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>月租卡</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>储值卡</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnCancel">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>620</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>储值卡余额提示</string>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>128</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>剩余</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>元时提醒</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>格式化脱机数据</string>
     </property>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>开始格式化</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>强行开闸</string>
     </property>
     <widget class="QCheckBox" name="checkBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>禁止鼠标点击开闸</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>进入时需要确认放行</string>
     </property>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>月租卡</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>储值卡</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>计时卡</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>月租卡工作模式</string>
     </property>
     <widget class="QCheckBox" name="checkBox_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>允许多进多出</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>费用计算设置</string>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>使用自定义设置</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>使用外接DLL文件</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnCommonSave">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="tabUseTimeLimit">
   <attribute name="title">
    <string>使用时间限制</string>
   </attribute>
   <widget class="QGroupBox" name="gbTimeLimit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>511</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>软件有效期设置</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>471</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>当软件超过此处有效设置的时间后将不能正常使用，若非特殊原请在此处设置较大日期</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOK">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateValid">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>90</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="tabVideoModeSetting">
   <attribute name="title">
    <string>视频模式设置</string>
   </attribute>
   <widget class="QWidget" name="wgVideo" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>201</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnVideoModeSave">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>150</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>视频数目设置</string>
     </property>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>2路</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>3路</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>4路</string>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>RGB24  320 X 240</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RGB24  640 X 480</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RGB24  720 X 288</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RGB24  720 X 576</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="chkStartVideo">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>启动视频</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="tabVehicleIDSetting">
   <attribute name="title">
    <string>车牌自动识别设置</string>
   </attribute>
   <widget class="QWidget" name="wgVichle" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>341</width>
      <height>351</height>
     </rect>
    </property>
    <widget class="QCheckBox" name="checkBox_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>启动车牌识别功能</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>151</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>识别精度设置</string>
     </property>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>不允许误差</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>允许误差一位</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>允许误差三位</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>允许误差二位</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>允许误差六位</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>允许误差七位</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>允许误差五位</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>允许误差四位</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_24">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>允许误差八位</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnVichle">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>300</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="tabSpecialSetting">
   <attribute name="title">
    <string>特殊设置</string>
   </attribute>
   <widget class="QWidget" name="wgSpecial" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>631</width>
      <height>231</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>车位计算</string>
     </property>
     <widget class="QCheckBox" name="checkBox_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>月租卡不计入车位</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>储值卡不计入车位</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnSpecialSave">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>181</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>数据同步设置</string>
     </property>
     <widget class="QCheckBox" name="checkBox_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>不同步脱机数据</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_16">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>271</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>中心控制器刷卡设置</string>
     </property>
     <widget class="QCheckBox" name="checkBox_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>允许代替出场卡使用</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>允许代替入场刷卡使用</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_17">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>271</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>储值卡计费</string>
     </property>
     <widget class="QCheckBox" name="checkBox_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>储值卡24小时内不重复使用</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
